### ISSUE TITLE
Kolbot Log In Issue

### ISSUE THREAD

**Issue by InZSan3** [kolton/d2bot-with-kolbot#886]
Hello guys.
Im new to Kol Bot and i did everything to setup like it was explained in a youtube Video. But when i start the bot he tries to login and "Diablo" tells me that the password is incorrect. But i definitely use correct account name and password in my Bot Setup. 
Can someone help me with that ?

**Comment by 5noop** [comment]
Did you setup the correct realm ?

**Comment by InZSan3** [comment]
Yes i also setup the correct realm. Here a picture of my profile... maybe u guys got an idea of what could be wrong here
![botscreen](https://user-images.githubusercontent.com/46272539/50560655-4712fb00-0d03-11e9-8aef-e2c82984e9a8.JPG)

**Comment by semirotta** [comment]
To my eyes that looks all ok, most likely some kind of configuration issue perhaps instead in the files

**Comment by InZSan3** [comment]
No there is only letters and numbers in Name and Password

**Comment by 5noop** [comment]
Open profile.json (in data folder) and check if you have the right password in it.

**Comment by InZSan3** [comment]
Everything seems correct in there! I Really dont know why this sh it isnt working -.-

**Comment by InZSan3** [comment]
Okay maybe i found the problem. I installed Diablo 2 from blizzard site... do i need any other version maybe ? If yes can someone link me the working version ?

**Comment by InZSan3** [comment]
Okay i solved the problem. Its not a problem of the bot but was a not known fact by me. Turned out my password is 13 letters long, while diablo 2 only takes 12 letters long passwords. I never realized this, cause i alwas typed in the whole 13 letters password while the last letter was just never filled into the password field. The Bot seemed to copy paste the 13 letter password into the field and this caused problems. I just gave the bot the password without my last letter and he managed to log in!

**Comment by semirotta** [comment]
Ah that was bit annoying to figure out. Good to know in the future for everyone else aswell.
